daemon/api.go

```diff
@@ -2519,6 +2519,17 @@ func postDebug(c *Command, r *http.Request, user *auth.UserState) Response {
 		}, nil)
 	case "can-manage-refreshes":
 		return SyncResponse(devicestate.CanManageRefreshes(st), nil)
+	case "connectivity":
+		// FIXME: add to snapstate.StoreService? feels not ideal :/
+		//        other options?
+		s := snapstate.Store(st).(*store.Store)
+		st.Unlock()
+		connectivity, err := s.ConnectivityCheck()
+		st.Lock()
+		if err != nil {
+			return InternalError("cannot run connectivity check: %v", err)
+		}
+		return SyncResponse(connectivity, nil)
```

Comment:
I think the new discussion plan was to hide domains also here, so something like

{"ok": true} if everything is fine
{"unreachable": [domains]}  with the ones that are false

or something like that

Reply:
Oh, I thought we just hide it in the UI. I can hide it at this level as well.

Reply:
I only return the unreachable machines via the API now.